### PR TITLE
CONCF-296 add siteMessage to meta tags

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/CONCF-296

This is part of the ticket. Second part can be found here: https://github.com/Wikia/app/pull/6697

### CODE REVIEW

**Comment by bognix** [comment]
LGTM
ping @hakubo

**Comment by hakubo** [comment]
LGTM

File: server/views/layout.hbs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 		<meta charset="utf-8">
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
 		<title>{{displayTitle}}{{#displayTitle}} - {{/displayTitle}}{{wiki.siteName}}</title>
-		<meta name="keywords" content="{{wiki.siteName}},{{wiki.dbName}},{{displayTitle}}">
+		<meta name="keywords" content="{{wiki.siteMessage}},{{wiki.siteName}},{{wiki.dbName}},{{displayTitle}}">
```
Comment (rogatty) [inline server/views/layout.hbs]: JIRA issue is about _the community name at the top of the page_ and not about keywords, right?

**Comment by bognix** [comment]
ping @hakubo @dianafa

**Comment by dianafa** [comment]
LGTM

File: server/views/layout.hbs [hunk 2]
```diff
@@ -4,7 +4,7 @@
 		<meta charset="utf-8">
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
 		<title>{{displayTitle}}{{#displayTitle}} - {{/displayTitle}}{{wiki.siteName}}</title>
-		<meta name="keywords" content="{{wiki.siteName}},{{wiki.dbName}},{{displayTitle}}">
+		<meta name="keywords" content="{{#wiki.siteMessage}}{{wiki.siteMessage}},{{/wiki.siteMessage}}{{wiki.siteName}},{{wiki.dbName}},{{displayTitle}}">
```
Comment (Warkot) [inline server/views/layout.hbs]: I would make this line and the one above it consistent. If understand what it's doing correctly.
```{{displayTitle}}{{#displayTitle}} - {{/displayTitle}}```
Print the ` - ` only if there's a `displayTitle`, right? And I feel your solution is better as it checks the value of the `wiki.siteMessage` earlier and ignores whole block if it's falsy.

**Comment by tomasznapieralski** [comment]
Keywords need fixes.
- additional coma (when message is empty)
- siteName and displayTitle are the same

**Comment by bognix** [comment]
ping @dianafa @Warkot

**Comment by dianafa** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM :shipit:

**Comment by tomasznapieralski** [comment]
LGTM

File: server/views/layout.hbs [hunk 3]
```diff
@@ -3,8 +3,8 @@
 	<head>
 		<meta charset="utf-8">
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
-		<title>{{displayTitle}}{{#displayTitle}} - {{/displayTitle}}{{wiki.siteName}}</title>
-		<meta name="keywords" content="{{wiki.siteName}},{{wiki.dbName}},{{displayTitle}}">
+		<title>{{#if displayTitle}}{{displayTitle}} - {{/if}}{{wiki.siteName}}</title>
```
Comment (rogatty) [inline server/views/layout.hbs]: @dianafa @bognix:
>Due to the potential for SEO-focused customization (and some existing customizations), SITENAME_MSG_KEY (MediaWiki:pagetitle) is only appropriate for use in the meta tags.

>siteName, as set in the WF variable, should be used for showing the wikia's name in the page layout

Shouldn't we use `siteMessage` in the title too? It's a meta tag, right?

Sorry for the late response :see_no_evil:

Comment (dianafa) [inline server/views/layout.hbs]: What for? For SEO?

Generally the siteMessage is customizable and can be ugly. I don't think we want to have set of strange for-seo-purposes-only words in <title>. Having them in meta seem sufficient for me.

Comment (bognix) [inline server/views/layout.hbs]: @rogatty - see the comments in CONCF-296. Implemented solution is what Bert proposed and what he agreed to (see last comment)

**Pull request merged**